Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
+
+## Table on Contents
+- [TL;DR](#tldr)
+- [Projects](#projects)
+- [Roles](#roles)
+  - [Product Owners](#product-owners)
+  - [Project Leads](#project-leads)
+  - [Project Managers](#project-managers)
+- [Structure](#structure)
+  - [Overview](#overview)
+  - [Goals](#goals)
+  - [Backlog](#backlog)
+  - [Milestones](#milestones)
+  - [Project Roadmap](#project-roadmap)
+  - [Organization Roadmap](#organization-roadmap)
+- [Flow](#flow)
+  - [Quarterly Planning](#quarterly-planning)
+  - [Software Development Pipeline](#software-development-pipeline)
+    - [Kanban Pipeline](#kanban-pipeline)
+- [Implementation](#implementation)
+
+## TL;DR
+
+The IPFS project management process has the following **cadence**:
+
+Roadmap Updates
+- every 3 months
+- identify, discuss and plan the next steps for the organization as a whole
+
+"Sprint" Updates
+- once a week (2 weeks?)
+- discuss last week's progress
+- discuss plan for the next week
+
+The IPFS project management process breaks down to the following **units of work**:
+
+Projects 
+- may contain other projects
+- have a project leader who "roadmaps" (decides what are goals and milestones, and when they should happen)
+- have a roadmap (list of milestones and goals)
+- have a backlog (list of goals)
+
+Milestones 
+- measurable unit of progress
+- contains a list of goals, ordered by priority
+- usually but not always sequentially ordered
+- may have a deadline
+
+Goals
+- an individual actionable unit of work
+- may have other Goals as dependencies
+- may have parts or sub-goals
+- may have parts or sub-goals big enough to merit own Goals
+
+The IPFS project management process tracks the work with the following **tools**:
+
+Roadmaps
+- attached to a project
+- has a list of milestones, ordered by completion time
+
+Pipelines
+- are stages through which goals move, the steps between "not done" and "done"
+- guide the software development process
```

Comment:
I'm not sure they do. They're just labels, if what I read above is right. Can we be clearer on what stages there are? How does that guide the process?